src/main/groovy/io/seqera/wave/service/persistence/impl/SurrealPersistenceService.groovy

```diff
@@ -182,6 +182,21 @@ class SurrealPersistenceService implements PersistenceService {
         return result
     }
 
+    @Override
+    List<WaveBuildRecord> allBuilds(String containerId) {
+        final query = """
+            select * 
+            from wave_build 
+            where buildId ~ '${containerId}${BuildRequest.SEP}'
+            order by buildId
```

Comment:
Better to order by time, the id could not be consist when passing from `_9` to `_10` and similar

Reply:
done